src/components/RoomDropdown/MemberOptions.tsx

```diff
@@ -0,0 +1,228 @@
+    min-height: 85px;
+    background-color: #f1f4f7;
+    margin: 5px;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+`
+
+const Subtitle = styled.span`
+    font-size: 12px;
+    color: #59799c;
+
+    cursor: pointer;
+`
+
+const MemberIcon = styled.div`
+    height: 3rem;
+    width: 3rem;
+    background-color: #cdcfd1;
+    border-radius: 50%;
+    margin: 0px 20px;
+`
+
+const EditButton = styled.div`
+    position: relative;
+    height: 2.5rem;
+    width: 2.5rem;
+    background-color: white;
+    border-radius: 50%;
+    margin: 0px 20px;
+    margin-left: auto;
+`
+
+const MemberName = styled.span`
+    width: 224px;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    line-height: 30px;
+`
+
+const MemberInput = styled.input`
+    border: 1px solid #dee6ee;
+    border-radius: 8px;
+    font-size: 16px;
+    width: 100%;
+    line-height: 18px;
+    padding: 7px 12px;
+`
+
+const DropDownContainer = styled.div`
+    margin-left: 10px;
+`
+
+const MemberContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`
+
+const DropDownHeader = styled.div`
+    margin-bottom: 0.8em;
+    padding: 0 2em 0 1em;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.15);
+    font-weight: 500;
+    font-size: 1.3rem;
+    color: #3faffa;
+    background: #ffffff;
+`
+const DropDownListContainer = styled.div`
+    position: absolute;
+    top: 45px;
+    right: 0px;
+    z-index: 100;
+`
+
+const DropDownList = styled.div`
+    width: fit-content;
+    padding: 0;
+    margin: 0;
+    background: #ffffff;
+    border-radius: 10px;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+    box-sizing: border-box;
+    color: #59799c;
+    font-family: ${KARELIA};
+    font-size: 14px;
+    list-style-type: none;
+    border-spacing: 0px;
+
+    hr {
+        border: 0;
+        border-top: 1px solid #dee6ee;
+        padding: 0px;
+        margin: 0px;
+    }
+`
+
+const ListItem = styled.div`
+    white-space: nowrap;
+    padding: 10px 15px;
+    background-clip: padding-box;
+    display: flex;
+    align-items: center;
+
+    &:first-child {
+        padding-top: 15px;
+        border-radius: 10px 10px 0 0;
+    }
+
+    &:last-child {
+        padding-bottom: 15px;
+        border-radius: 0 0 10px 10px;
+    }
+
+    :hover {
+        background-color: #f1f4f7;
+    }
+
+    svg {
+        width: 1rem;
+        margin-right: 0.75rem;
+    }
+`
+
+const UnstyledMemberOptions = ({ address }: any) => {
+    const [isDropdownOpen, setDropdownOpen] = useState<boolean>(false)
+    const [editing, setEditing] = useState<boolean>(false)
+    const [nickname, setNickname] = useState<string>('')
+
+    const ref = useRef<HTMLDivElement>(null)
+    const buttonRef = useRef<HTMLDivElement>(null)
+
+    useEffect(() => {
+        function handleClickOutside(event: any) {
+            setDropdownOpen(false)
+        }
+
+        // Bind the event listener
```

Comment:
This comment looks redundant. The following line is exactly what it says it's doing. Failry obvious.